pkg/kernel/interface.go

```diff
@@ -93,6 +93,15 @@ func (intf *Interface) AddLocalPrefix(prefix string) error {
 	if err != nil {
 		return err
 	}
+	addrList, err := netlink.AddrList(i, netlink.FAMILY_ALL)
```

Comment:
This is a race. Probably never happens, but the address can in theory be added after the check. IMO it would be better to check the err and ignore any "already exist" error.

Reply:
I also think checking for "already exist" error would be more straight forward.

Reply:
The error is a common error (syscall.Errno=file exists), if I ignore it, it will ignore all errors

Reply:
Sorry, I am not sure what I tried yesterday, but it works with the fix you proposed works fine. I modified the commit